c++/src/RleDecoderV2.cc

```diff
@@ -67,6 +91,147 @@ namespace orc {
   }
 
   void RleDecoderV2::readLongs(int64_t* data, uint64_t offset, uint64_t len, uint64_t fbs) {
+    uint64_t startBit = 0;
+#if ENABLE_AVX512
```

Comment:
It would be better to be able to disable it at the runtime.

Reply:
Do you mean to create a new option in ORC config file to check the option "on" or "off" at the runtime? If yes, may I have a question that where is the ORC config file?

Reply:
I mean we can use an environment variable like `std::getenv("ENABLE_RUNTIME_AVX512")` as a toggle.